site/public/js/grade-inquiry.js

```diff
@@ -1,6 +1,27 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
```

Comment:
```suggestion
/* exported saveDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
```
To fix JS-lint, add `saveDraft` to the exported list.